modules/nf-core/busco/meta.yml

```
@@ -56,6 +56,14 @@ output:
       type: directory
       description: BUSCO lineage specific output
       pattern: "*-busco"
+  - full_table:
+      type: file
+      description: Full BUSCO results table
+      pattern: "*-busco/*/run_*/full_table.tsv"
+  - seq_dir:
+      type: dir
+      description: BUSCO sequence directory
+      pattern: "*-busco/*/run_*/busco_sequences"
```

Only the name needs to be documented
```suggestion
      pattern: "busco_sequences"
```